controller/src/main/java/org/jboss/as/controller/Stream.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.as.controller;
+
+/**
+ * @author Paul Ferraro
+ */
+public enum Stream {
```

Comment:
This is just kind of a general though. Should we use another name besides stream because of `java.util.stream.Stream`?

Reply:
Honestly, I could think of a better name. Any ideas?

Reply:
ModelStream?

Reply:
ModelStream sounds good to me.

Reply:
I've updated this to FeatureStream.  If anyone has a better idea, let me know.